transport/wsn/src/main/java/org/kaazing/gateway/transport/wsn/WsnConnector.java

```diff
@@ -581,8 +580,6 @@ public WsnSession call() throws Exception {
                         parent.setAttribute(ENCODING_KEY, Encoding.BINARY);
                     } else if ("base64".equals(frameType)) {
                         parent.setAttribute(ENCODING_KEY, Encoding.BASE64);
-                    } else {
-                        parent.setAttribute(ENCODING_KEY, Encoding.TEXT);
```

Comment:
Not sure why connector was sending text messages by default. I don't know if there are any backward compatible issues here by this change